### PR TITLE
Change default hostname based off of ipv6 support

### PR DESCRIPTION
**Pull request by grouma**
Closes https://github.com/dart-lang/webdev/issues/233

### CODE REVIEW

File: webdev/lib/src/command/configuration.dart [hunk 1]
```diff
@@ -121,9 +124,36 @@ class Configuration {
...
+    String hostname,
+    bool launchInChrome,
+    bool logRequests,
+    String output,
+    ReloadConfiguration reload,
+    bool release,
+    bool requireBuildWebCompilers,
+    bool verbose,
+  }) async {
+    hostname ??= await supportsIpv6 ? '0.0.0.0' : 'localhost';
```
Comment (kevmoo) [inline webdev/lib/src/command/configuration.dart]: Isn't `0.0.0.0` "listen ALL"? This seems a bad idea. That means, by default, you'll be able to connect to the machine from anywhere (assuming no firewall).

Maybe use the v6 loopback? `::1/128` - https://en.wikipedia.org/wiki/IPv6_address#Local_addresses

File: dwds/pubspec.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ description: >-
 environment:
   sdk: ^2.1.0
 dependencies:
+  http_multi_server: ^2.0.0
```
Comment (kevmoo) [inline dwds/pubspec.yaml]: Even better!

File: webdev/lib/src/serve/chrome.dart [hunk 3]
```diff
@@ -52,7 +52,7 @@ class Chrome {
     await _dataDir?.delete(recursive: true);
   }
 
-  /// Connects to an instance of the Chreom.
+  /// Connects to an instance of the Chrome.
```
Comment (natebosch) [inline webdev/lib/src/serve/chrome.dart]: This feels like an incomplete thought to me. What is "the Chrome"?

Comment (grouma) [reply webdev/lib/src/serve/chrome.dart]: You know... like... the one and only... :p

Expanded

**Pull request merged**